doc/users/prev_whats_new/whats_new_3.3.0.rst

```diff
@@ -304,43 +425,180 @@ clabels has been changed to (``2 + zorder`` passed to `~.axes.Axes.contour` /
 `~.axes.Axes.contourf`).
 
 
+Better control of ``Axes.pie`` normalization
+--------------------------------------------
+
+Previously, `.Axes.pie` would normalize its input *x* if ``sum(x) > 1``, but
+would do nothing if the sum were less than 1. This can be confusing, so an
+explicit keyword argument *normalize* has been added. By default, the old
+behavior is preserved.
+
+By passing *normalize*, one can explicitly control whether any rescaling takes
+place or whether partial pies should be created. If normalization is disabled,
+and ``sum(x) > 1``, then an error is raised.
+
+.. plot::
+
+    def label(x):
+        return [str(v) for v in x]
+
+    x = np.array([0.25, 0.3, 0.3])
+    fig, ax = plt.subplots(2, 2, constrained_layout=True)
+
+    ax[0, 0].pie(x, autopct='%1.1f%%', labels=label(x), normalize=False)
+    ax[0, 0].set_title('normalize=False')
+    ax[0, 1].pie(x, autopct='%1.2f%%', labels=label(x), normalize=True)
+    ax[0, 1].set_title('normalize=True')
+
+    # For the purposes of keeping the documentation build warning-free, and
+    # future proof for when the deprecation is made permanent, we pass
+    # *normalize* here explicitly anyway.
+    ax[1, 0].pie(x, autopct='%1.2f%%', labels=label(x), normalize=False)
+    ax[1, 0].set_title('normalize unspecified\nsum(x) < 1')
+    ax[1, 1].pie(x * 10, autopct='%1.2f%%', labels=label(x * 10),
+                 normalize=True)
+    ax[1, 1].set_title('normalize unspecified\nsum(x) > 1')
+
```

Comment:
why does this need to be in the what's new?

Reply:
What part specifically?

Reply:
the code and particulary this note:
>
 For the purposes of keeping the documentation build warning-free, and
    # future proof for when the deprecation is made permanent, we pass
    # *normalize* here explicitly anyway.

Reply:
Sorry, I'm confused; didn't you [ask for code](https://github.com/matplotlib/matplotlib/pull/17858#discussion_r452600178)?

Reply:
😦 sorry for the confusion, I did ask for code and the note threw me 'cause I read it as explaining a deprecated feature rather than a new one. Maybe moveing the comment under the code could make it clearer?

Reply:
Oh, well this _is_ showing deprecated behaviour as comparison with the new explicit one, but because it's deprecated, I had to use the new explicit one in the code. It's labelled with the old way in the figure though. (Note, this code isn't shown by default.) I could remove it or change the comment description a little?

Reply:
Oh, if the code/comment isn't shown than it's good enough...